src/Particular.CodeRules/AwaitOrCaptureTasks/AwaitOrCaptureTasksAnalyzer.cs

```diff
@@ -28,16 +28,15 @@ public override void Initialize(AnalysisContext context)
             context.RegisterSyntaxNodeAction(AnalyzeSyntax, SyntaxKind.InvocationExpression);
         }
 
-        private static void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
+        static void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
         {
             var node = context.Node as InvocationExpressionSyntax;
-            var parentNode = node?.Parent as ExpressionStatementSyntax;
 
-            if (parentNode != null)
+            if (node?.Parent is ExpressionStatementSyntax _)
```

Comment:
I think you should be able to make this:
```suggestion
            if (node?.Parent is ExpressionStatementSyntax)
```

Reply:
👍 
![image](https://user-images.githubusercontent.com/1309622/106217702-09f3ee00-6193-11eb-973a-97050517ee25.png)